Classes/Response/YLPBaseObject.h

```diff
@@ -0,0 +1,15 @@
+//
+//  YLPBaseObject.h
+//  YelpAPI
+//
+//  Created by Brian Sharrief Alim Bowman on 11/7/16.
+//
+//
+
+@import Foundation;
+
+@interface YLPBaseObject : NSObject
+
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
Yeah that's a fair point, I guess I was thinking mainly that `YLPBaseObject` seems a little odd as a class with no functionality.

I'm debating whether it's worth it or if it'd be fine to just copy-paste this `+ (instancetype)new NS_UNAVAILABLE` line everywhere... the convenience of only having to write it once is pretty nice.